R/utils-get_code_dependency.R

```diff
@@ -64,7 +65,8 @@ get_code_dependency <- function(code, names, check_names = TRUE) {
 
   lib_ind <- detect_libraries(calls_pd)
 
-  as.character(code[sort(unique(c(lib_ind, ind)))])
+  code_ids <- sort(unique(c(lib_ind, ind)))
+  trimws(paste(as.character(code[code_ids]), comments[code_ids]))
```

Comment:
`teal.code::get_code_dependency` now returns comments as well, so that when it is used in `subset` it creates an object that does contain comments in the final `@code`.

We might edit `teal.code::get_code` so it drops comments, so this will not be shown in the final extracted code.
Check this PR that shows what is changed in the returned code https://github.com/insightsengineering/teal/pull/1396

Comments are included.